daml-script/export/src/main/scala/com/daml/script/export/Encode.scala

```diff
@@ -21,35 +21,40 @@ import spray.json._
 
 private[export] object Encode {
 
-  def encodeArgs(export: Export): JsObject = {
+  def encodeArgs(exporting: Export): JsObject = {
```

Comment:
Can we call it `scriptExport` instead? Seems a bit clearer.

Reply:
many thanks for the suggestion! Will apply that in an instant.